shaptools/hana.py

```diff
@@ -73,7 +78,8 @@ class HanaInstance(object):
     """
 
     PATH = '/usr/sap/{sid}/HDB{inst}/'
-    INSTALL_EXEC = '{software_path}/DATA_UNITS/HDB_LCM_{platform}/hdblcm'
+    INSTALL_EXEC = 'hdblcm'
+    HANA_PLATFORM = '^HDB:HANA:.*:{platform}:SAP HANA PLATFORM EDITION.*'
```

Comment:
@arbulu89 a small suggestion here is that maybe we should ignore the Commercial name here `SAP HANA PLATFORM EDITION` as this can be changed by SAP easier than the first two fields and the formula would brake because or that.

Reply:
Ok, I will only use '^HDB:HANA:.*:{platform}:.*'
In fact, in the future, if we start working with HANA 1, we might need to polish this to consider the version too